pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -67,6 +67,9 @@ stdenv.mkDerivation rec {
     glib
     polkit
     python3 # for cli tool
+    # Used only for check, but gobject-introspection doesn't populate
+    # GI_TYPELIB_PATH with checkInputs since https://github.com/NixOS/nixpkgs/pull/177538
```

Comment:
this is actually a manifestation of https://github.com/NixOS/nixpkgs/issues/161570 (checkInputs adds to nativeBuildInputs)
gobject-introspection actually does the right thing now so this issue was exposed